benchmark/performance_utils.py

```diff
@@ -65,39 +71,141 @@ def run(self):
 POINTWISE_BATCH = 1024
 REDUCTION_BATCH = 1024
 BLAS_BATCH = 16
-SIZES = [i * 64 for i in range(1, 21)]
+SIZES = [i * 64 for i in range(1, 22, 5)]
 
 
 def unary_arg(dtype, batch, size):
-    inp = torch.randn([batch, size], dtype=dtype, device="cuda")
+    inp = torch.randn([batch, size], dtype=dtype, device=DEVICE)
     return (inp,)
 
 
 def unary_int_arg(dtype, batch, size):
     inp = torch.randint(
-        low=0, high=0x7FFF, size=[batch, size], dtype=dtype, device="cuda"
-    )
+        low=0, high=0x7FFF, size=[batch, size], dtype=dtype, device="cpu"
+    ).to(DEVICE)
     return (inp,)
 
 
 def binary_args(dtype, batch, size):
-    inp1 = torch.randn([batch, size], dtype=dtype, device="cuda")
-    inp2 = torch.randn([batch, size], dtype=dtype, device="cuda")
+    inp1 = torch.randn([batch, size], dtype=dtype, device=DEVICE)
+    inp2 = torch.randn([batch, size], dtype=dtype, device=DEVICE)
     return inp1, inp2
 
 
 def binary_int_args(dtype, batch, size):
     inp1 = torch.randint(
-        low=0, high=0x7FFF, size=[batch, size], dtype=dtype, device="cuda"
-    )
+        low=0, high=0x7FFF, size=[batch, size], dtype=dtype, device="cpu"
+    ).to(DEVICE)
     inp2 = torch.randint(
-        low=0, high=0x7FFF, size=[batch, size], dtype=dtype, device="cuda"
-    )
+        low=0, high=0x7FFF, size=[batch, size], dtype=dtype, device="cpu"
+    ).to(DEVICE)
     return inp1, inp2
 
 
 def ternary_args(dtype, batch, size):
-    inp1 = torch.randn([batch, size], dtype=dtype, device="cuda")
-    inp2 = torch.randn([batch, size], dtype=dtype, device="cuda")
-    inp3 = torch.randn([batch, size], dtype=dtype, device="cuda")
+    inp1 = torch.randn([batch, size], dtype=dtype, device=DEVICE)
+    inp2 = torch.randn([batch, size], dtype=dtype, device=DEVICE)
+    inp3 = torch.randn([batch, size], dtype=dtype, device=DEVICE)
     return inp1, inp2, inp3
+
+
+def cross_entropy_loss_args(dtype, batch, size):
```

Comment:
function cross_entropy_loss_args, cumsum_args and so on have been implemented in the corresponding test functions. they could be deleted here.

Reply:
ok, will delete in https://github.com/FlagOpen/FlagGems/pull/80